pkg/clusterversion/cockroach_versions.go

```diff
@@ -542,6 +542,9 @@ const (
 	// that (optionally) embed below-raft admission data.
 	V23_2_UseACRaftEntryEntryEncodings
 
+	// V23_2_PGLogicalReplication gates the use of PGLogicalReplication features.
+	V23_2_PGLogicalReplication
```

Comment:
nit: no need to make a new cluster version. you can just use `clusterversion.V23_2`

Reply:
there is no `V23_2` though, only `V23_2Start`.

Reply:
oh it's not in the struct